opentelemetry-sdk/src/opentelemetry/sdk/trace/sampling.py

```diff
@@ -113,11 +114,11 @@ class Sampler(abc.ABC):
     @abc.abstractmethod
     def should_sample(
         self,
-        parent_span_context: Optional["SpanContext"],
+        parent_context: Optional["Context"],
         trace_id: int,
         name: str,
         attributes: Attributes = None,
-        links: Sequence["Link"] = (),
+        links: Sequence["Link"] = None,
```

Comment:
Was this change to the default value of `links` used anywhere in this change?

Reply:
Not relevant to this change. Just design-wise we shouldn't have default values for mutable parameters.